index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+     <title> Portfolio</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+
+<body>
+
+<header>
+
+    <ul class="container1">
+    <li class="logo">Welcome</li>
+    <li class="item-center"> </li>
```

Comment:
- [ ] I suggest you remove this element entirely and use a CSS propery `justify-content: space-between;' for the parent element 💡